src/postmate.js

```diff
@@ -106,7 +121,7 @@ class ParentAPI {
 
     this.listener = (e) => {
       const { data, name } = (((e || {}).data || {}).value || {});
-      if (e.data.postmate === 'emit') {
+      if (e.data.postmate === 'emit' && e.data.childId === this.childId) {
```

Comment:
Could `e.data.childId === this.childId` be cached. (looking at line 142 in this PR)